7_react/react1/week2/todo-app/src/todoList/TodoList.jsx

```diff
@@ -0,0 +1,53 @@
+import TimeCount from "../timer/Timer";
+import TodoItem from "../todoItem/TodoItem";
+import todos from "../todoItem/todos";
+import { useState } from "react";
+
+function AppTodo() {
+  // const data = [...todos];
+  const [todolist, setTodoList] = useState([...todos]);
+
+  const addTodo = () => {
+    const newTodo = {id:new Date(), name: "new todo", date: new Date(), status: false };
```

Comment:
You are never using the date value from this object. Did you want to name it 'deadline' as in your initial todo array? If so, also note that new Date() returns a date object, so you would need to turn it into a string using [`.toLocaleDateString()` function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString)